torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14301,8 +14304,10 @@ def ref_pairwise_distance(input1, input2):
            supports_fwgrad_bwgrad=True,
            # https://github.com/pytorch/pytorch/issues/66357
            check_batched_forward_grad=False,
-           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           dtypes=all_types_and_complex_and(torch.complex32, torch.bool, torch.float16, torch.bfloat16),
+           backward_dtypes=floating_and_complex_types_and(torch.float16, torch.bfloat16),
```

Comment:
This will start failing once we support `sum` for `complex32` (as it is used in the backward)